### PR TITLE
Micosoft.OpenJDK.17 version 17.35.1

### PR DESCRIPTION
**Pull request by d3r3kk**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28974)

### CODE REVIEW

**Comment by d3r3kk** [comment]
For sanity checking: @karianna, @brunoborges, and @gdams. Thanks!

File: manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Publisher: Microsoft
+PublisherUrl: https://www.microsoft.com
+PublisherSupportUrl: https://support.microsoft.com
+PrivacyUrl: https://privacy.microsoft.com/en-us/privacystatement
+Author: Microsoft
+PackageName: Microsoft Build of OpenJDK with Hotspot 17+35 (x64)
+PackageUrl: https://www.microsoft.com/openjdk
+License: GPL2 with Classpath Exception
+LicenseUrl: https://openjdk.java.net/legal/gplv2+ce.html
+ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft"s new way to collaborate and contribute to the Java ecosystem.
```
Comment (jedieaston) [inline manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml:15]:
```suggestion
ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.
```

I'm 80% sure this is my fault, but it still needs to be fixed :-)

**Review by jedieaston — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml (hunk 1 above)
Comment (d3r3kk) [reply manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml]: No blame game here, friend! 😅

Comment (d3r3kk) [reply manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml]: (Couldn't repair the older manifests in this PR 🤦‍♂️)

**Review by jedieaston — APPROVED** [review]
Thanks!

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**